src/graphql/admin/root/mutation/business-update-map-info.ts

```diff
@@ -37,23 +36,27 @@ const BusinessUpdateMapInfoMutation = GT.Field({
       }
     }
 
-    const user = await updateBusinessMapInfo({ username, title, latitude, longitude })
+    const coordinates = {
+      latitude,
+      longitude,
+    }
+
+    const account = await updateBusinessMapInfo({ username, title, coordinates })
 
-    if (user instanceof Error) {
-      return { errors: [{ message: user.message }] }
+    if (account instanceof Error) {
+      return { errors: [{ message: mapError(account).message }] }
     }
 
     return {
       errors: [],
-      userDetails: {
-        id: user.id,
-        username: user.username,
-        level: user.level,
-        status: user.status,
-        phone: user.phone,
-        title: user.title,
-        coordinate: user.coordinate,
-        created_at: user.created_at,
+      accountDetails: {
+        id: account.id,
+        username: account.username,
+        level: account.level,
+        status: account.status,
+        title: account.title,
+        coordinates: account.coordinates,
+        createdAt: account.createdAt,
```

Comment:
if the attributes are the same you dont need to do this `accountDetails: account` is enough

Reply:
good point